src/styles/main.scss

```diff
@@ -1,3 +1,5 @@
-body {
-  margin: 0;
-}
+@import "blocks/variable.scss";
```

Comment:
Consider renaming 'blocks/variable.scss' to 'blocks/variables.scss' for better clarity.